.github/workflows/plugin-compiler-build.yml

```diff
@@ -0,0 +1,105 @@
+---
+name: Plugin-compiler build
+
+on:
+  workflow_run:
+    workflows:
+      - Release
+    types:
+      - completed
+
+env:
+  GOLANG_CROSS: 1.19-bullseye
+
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ fromJson(steps.verify-tag.outputs.result) && fromJson(steps.verify-tag.outputs.result).tag }}
```

Comment:
Is there docs for this, what does this do?

Reply:
This is rewritten now, but essentially this sets the output of this job to a json key(valid semver tag) returned from a github-script script in one of the steps.